libs/storage/include/storage/random_access_stack.hpp

```diff
@@ -186,11 +186,11 @@ class RandomAccessStack
     }
 
     // Get length of file
-    file_handle_.seekg(0, file_handle_.end);
+    file_handle_.seekg(0, decltype(file_handle_)::end);
```

Comment:
Seems like these should simply just `std::ios:: end` etc

Reply:
done